### ISSUE TITLE
Checksums for the webfont downloads (free)

### ISSUE THREAD

**Issue by ChosenSilver** [FortAwesome/Font-Awesome#19039]
Is there any way that anyone knows of to get the checksums for the zipped files? I need to bring these files into a secured environment and I am not able to do that without being able to validate that the checksum on the zip file matches what FontAwesome says it should be. I already emailed support and they sent me here.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

You mean zip files available at https://fontawesome.com/download and https://github.com/FortAwesome/Font-Awesome/releases/tag/6.1.1 ?

**Comment by ChosenSilver** [comment]
Yes, exactly. Specifically, it is fontawesome-free-5.10.1-web I am looking for

**Comment by tagliala** [comment]
Thanks for the clarification

@robmadole I think this is a legit request and a good practice for "binary" files.

I see that the release description is used for checksums, and in the future we may upload the checksums as part of the release itself ([example](https://github.com/wkhtmltopdf/wkhtmltopdf/releases)) and as per the website we have full control.

Can't tell which is the effort to automate this

**Comment by robmadole** [comment]
Yeah this is a great idea. I don't have an ETA on this but we'll get it on the todo list.

**Comment by ChosenSilver** [comment]
In the meanwhile, is there any way I can be provided with what the hash is supposed to be for the version I mentioned? I just need to be able to prove that the hash on the file I have matches what the source says it should be.

**Comment by ChosenSilver** [comment]
@tagliala Is there any way for me to get the hash in the meanwhile manually?
